src/components/dashboard/NotePad/NotePad.css

```diff
@@ -1,13 +1,15 @@
 .NotePad__root {
   width: 100%;
-  border: 1.5px solid var(--color-box-shadow);
   font-family: inherit;
   border-radius: 4px;
   position: relative;
   min-height: 10rem;
   border-radius: 4px;
```

Comment:
You can stack css based on order first display position width and then padding border